util/dockerfiles/devcontainer/Dockerfile

```diff
@@ -44,24 +44,24 @@ FROM ghcr.io/gem5/ubuntu-24.04_all-dependencies:latest AS builder
 LABEL org.opencontainers.image.source=https://github.com/gem5/gem5
 LABEL org.opencontainers.image.licenses=BSD-3-Clause
 
-RUN wget https://github.com/gem5/gem5/archive/refs/tags/v24.1.0.1.tar.gz && \
-    tar xzf v24.1.0.1.tar.gz && mv gem5-24.1.0.1 gem5 && rm v24.1.0.1.tar.gz
+RUN wget --no-check-certificate https://github.com/gem5/gem5/archive/refs/tags/v24.1.0.3.tar.gz && \
+    tar xzf v24.1.0.3.tar.gz && mv gem5-24.1.0.3 gem5 && rm v24.1.0.3.tar.gz
 WORKDIR /gem5
-RUN scons build/ALL/gem5.opt -j`nproc`
+RUN scons build/ALL/gem5.fast -j`nproc`
```

Comment:
I think the devcontainer should have opt... it is the *dev*container meant for *developing* with gem5. Specifically, when I use this in classes, it is useful to be able to enable debug flags.